capstone/src/main/java/com/google/sps/servlets/DashboardHandlerServlet.java

```diff
@@ -15,6 +24,20 @@
 @WebServlet("/dashboard-handler")
 public class DashboardHandlerServlet extends HttpServlet {
 
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery results = datastore.prepare(new Query(Classroom.CLASSROOM_ENTITY_NAME));
```

Comment:
This will get every classroom, even if the user is not in them? (Fine for now but I recommend leaving a TODO or note saying as such.)

Reply:
Yeah, until we get the web filter, we will query all of the classrooms for now.

Reply:
TODO comment added